canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -405,22 +416,45 @@ class ToolTip extends React.Component {
 		if (this.props.className) {
 			tipClass += " " + this.props.className;
 		}
-
+		let linkClicked = false;
 		let link = null;
 		if (this.state.isTooltipVisible && this.props.tooltipLinkHandler && this.props.link) {
 			const linkInformation = this.props.tooltipLinkHandler(this.props.link);
 			// Verify tooltipLinkHandler returns object in correct format
 			if (typeof linkInformation === "object" && linkInformation.label && linkInformation.url) {
-				link = (<Link
-					className="tooltip-link"
-					id={this.props.link.id}
-					href={linkInformation.url}
-					target="_blank"
-					rel="noopener"
-					visited={false}
+				link = (<div
+					onKeyDown={(evt) => {
+						evt.stopPropagation();
+						evt.preventDefault();
+						if (evt.key === "Escape") {
+							this.triggerRef.focus();
+							this.setTooltipVisible(false);
+						}
```

Comment:
When link is in focus, clicking `Enter` (windows) or `return`(mac) key should open the tooltip URL. 
Rest of the code looks good.

Reply:
Hi Neha. Sure, I have included above changes. Please review.